applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -139,8 +98,7 @@ static int audio_stream_index_get(struct bt_audio_stream *stream, uint8_t *index
 static bool all_active_headsets_started(void)
 {
 	for (int i = 0; i < ARRAY_SIZE(headsets); i++) {
-		if (headsets[i].sink_stream != NULL &&
-		    headsets[i].sink_stream->ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
+		if (headsets[i].sink_stream.ep->status.state != BT_AUDIO_EP_STATE_STREAMING) {
```

Comment:
Why are you removing the first check? Wouldn't that cause segmentation fault if sink_stream == NULL?

Reply:
Sink stream can never be NULL, but ep can be NULL.
I created a new function: ep_state_check()